db/log_format.h

```diff
@@ -41,10 +41,14 @@ enum RecordType : uint8_t {
   // User-defined timestamp sizes
   kUserDefinedTimestampSizeType = 10,
   kRecyclableUserDefinedTimestampSizeType = 11,
+
+  // For WAL verification
+  kPredecessorWALInfoType = 129,
```

Comment:
Let's try to set (and note in a comment) a precedent that for all the values >= 10, the 1 bit just indicates/toggles whether it's recyclable. For that we should use either 128, 129 or 130, 131 for these two enums, not 129, 130.

I think we could use that for some code simplification in the future.

Reply:
Fixed